evalml/tests/data_checks_tests/test_highly_null_data_check.py

```diff
@@ -155,8 +177,84 @@ def test_highly_null_data_check_warnings():
     }
 
 
+def test_highly_null_data_check_separate_rows_cols():
```

Comment:
Perhaps we want to adopt a similar testing strategy where we create no-null rows, all null rows and some null rows and just test that?  It's not immediately clear what this test is setting out to accomplish and it looks a lot like the other tests.

Reply:
Could you expand more about what you're thinking of? The previous test checks both null rows and columns at thresholds of 1, 0.5, and 0. This added test just checks that having different values for the row vs column thresholds will throw/not throw warnings correctly, I'm not quite sure why we'd need to add more explicit null row coverage.

Reply:
Ok, I didn't understand what the purpose of this specific test was.  Good to go.